.github/workflows/changelog.yml

```diff
@@ -0,0 +1,51 @@
+name: Changelog
+
+on:
+  push:
+    branches:
+      # todo: change this branch name
+      - master
+    repository_dispatch:
+      types: [semantic-release]
+
+env:
+  THIRD_PARTY_GIT_AUTHOR_EMAIL: opensource+bot@newrelic.com
+  THIRD_PARTY_GIT_AUTHOR_NAME: nr-opensource-bot
```

Comment:
It doesn't look like you're using these environment variables in this action, so feel free to delete these.